### ISSUE TITLE
Android not initializing correctly RN 0.63

### ISSUE THREAD

**Issue by mljlynch** [BranchMetrics/react-native-branch-deep-linking-attribution#706]
Hi I'm trying to set up `react-native-branch`. When I run the app, my dashboard says my android SDK is set up, but I can not create a `branchUniversalObject` from the android simulator. I can from the iOS simulator. My configuration code is below:

`Android.XML`
```
      <!-- Add deep link scheme and Branch links -->
      <intent-filter >
        <action android:name="android.intent.action.VIEW" android:host="open" />
        <category android:name="android.intent.category.DEFAULT" />
        <category android:name="android.intent.category.BROWSABLE" />
        <data android:scheme="******" />
      </intent-filter>

                <!-- Branch App Links -->
      <intent-filter android:autoVerify="true">
        <action android:name="android.intent.action.VIEW" />
        <category android:name="android.intent.category.DEFAULT" />
        <category android:name="android.intent.category.BROWSABLE" />
        <data android:scheme="https" android:host="***.app-test.link" />
        <data android:scheme="http" android:host="**.app-test.link" />
      </intent-filter>
      
      //
      
          <meta-data android:name="io.branch.sdk.BranchKey" android:value="key_test_***"/>
    <meta-data android:name="io.branch.sdk.BranchKey.test" android:value="key_test_***"/>
```
`MainActivity.java`  

```java
// Override onStart for Branch.io
  @Override
  protected void onStart() {
      super.onStart();
      RNBranchModule.initSession(getIntent().getData(), this);
  }

  // Override onNewIntent for Branch.io
  @Override
  public void onNewIntent(Intent intent) {
      super.onNewIntent(intent);
    if (intent != null &&
          intent.hasExtra("branch_force_new_session") && 
          intent.getBooleanExtra("branch_force_new_session",false)) {
        RNBranchModule.onNewIntent(intent);
    }
  }
  ```
  
  `MainApplication.java`
  
  ```
    @Override
  public void onCreate() {
    super.onCreate();
    SoLoader.init(this, /* native exopackage */ false);
    RNBranchModule.getAutoInstance(this); 
    initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
  }
  ```

**Comment by mljlynch** [comment]
FWIW: I'm getting this log: `D RNBranch: Sent session start broadcast for null` where the URI is null and I'm unsure why

**Comment by jf-branch** [comment]
Hi @mljlynch I noticed you closed this out, were you able to resolve this issue?

**Comment by mljlynch** [comment]
Yes, there were two issues: 

1. I had misconfigured a key value which was preventing me from creating links
2. The onIntent override was causing the branch listener to not work in the way expected for android. I have opened a separate ticket for this here: https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/issues/707

As a general point of feedback, it's really hard to understand what is causing branch not to work properly. It would be helpful if there were more verbose error messages returned for the react-native SDK.
